src/shared/components/plots/PlotsTabUtils.tsx

```diff
@@ -1623,10 +1654,20 @@ export function getAxisDescription(
     },
     clinicalAttributeIdToClinicalAttribute: {
         [clinicalAttributeId: string]: ClinicalAttribute;
+    },
+    customAttributeIdToClinicalAttribute: {
+        [clinicalAttributeId: string]: ClinicalAttribute;
     }
 ) {
     let ret = '';
     switch (selection.dataType) {
+        case CUSTOM_ATTR_DATA_TYPE:
+            const customAttr =
+                clinicalAttributeIdToClinicalAttribute[selection.dataSourceId!];
```

Comment:
should this be `customAttributeIdToClinicalAttribute` instead?

Reply:
Changed